src/scss/override/_bootstrap.scss

```diff
@@ -76,3 +76,8 @@
 // Utility classes
 @import "bootstrap-sass/assets/stylesheets/bootstrap/utilities";
 @import "bootstrap-sass/assets/stylesheets/bootstrap/responsive-utilities";
+
+//Overrides
+.navbar-right > div:first-child {
```

Comment:
@codefactorymx, can you please move this override to `winstrap/src/scss/override/_navbar.scss`?